src/Sentry/IScopeObserver.cs

```diff
@@ -29,4 +29,9 @@ public interface IScopeObserver
     /// Sets the user information.
     /// </summary>
     void SetUser(SentryUser? user);
+
+    /// <summary>
+    /// Sets the current trace
+    /// </summary>
+    void SetTrace(SentryId traceId, SpanId parentSpanId);
```

Comment:
spam should be optional no?
if don't have a sample rate set for tracing, we still pass a trace i'd around to link errors together right?

TwP project (trace without performance)?

Reply:
The way tracing without performance works is it expects a `SpanId` to create a hierarchy. Since we're leveraging the `PropagationContext` it requires that `parent span ID`.